skin.css

```diff
@@ -1,13 +1,50 @@
+/* css-skin-prabesh */
+
+
 .highlight {
     background-color: #ddd;
 }
 
+/* common style for all buttons with class btn */
 button.btn {
+  font-size: 1.09em;
   border-radius: 3px;
+  color: #2A31BB;
+  background: transparent;
+  padding: 14px 61px;
   border-style: solid;
   border-width: 1px;
   font-weight: normal;
   line-height: 1.15;
   text-align: center;
   vertical-align: middle;
+  margin-left: 7px;
+  outline: none;
+}
+
+/* styles only applicable to buttons with class btn--primary */

```

Comment:
Comments like this might be going over the top a bit. Generally I would assume that readers of my code can read CSS, so they should be able to work out that it's a button with class `btn--primary`. I would only add a comment if I'm doing something surprising, that would explain why.